### PR TITLE
update performance flow

### PR DESCRIPTION
**Pull request by dele-lookman**

### CODE REVIEW

File: performance-test/blazemeter.yaml [hunk 1]
```diff
@@ -13,10 +13,10 @@ scenarios:
     variables:
       PAUSE: 15000
       scheme: https
-      host:
-      username:
-      password:
-      email:
+      host: ivy-web-pre.azure.defra.cloud
```
Comment (Jozzey) [inline performance-test/blazemeter.yaml]: Need to remove the values for these credentials as it's a public repo

host:
username:
password:
email:

File: performance-test/config.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ scenarios:
     variables:
       PAUSE: 15000
       scheme: https
-      host:
+      host: ivy-web-snd.azure.defra.cloud
```
Comment (Jozzey) [inline performance-test/config.yaml]: Should remove the host value

File: performance-test/blazemeter.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ execution:
     ramp-up: 1m
     hold-for: 12m
     locations:
-      eu-west-2: 1
+      eu-west-3: 1
```
Comment (Jozzey) [inline performance-test/blazemeter.yaml]: Should keep the location as eu-west-2 (London) rather than eu-west-3 (Paris)

**Review by Jozzey — CHANGES_REQUESTED** [review]
Couple of tweaks to do. Shouldn't really be adding any sensitive/personal info into the repo as it's public. Not a big for info that is in here as it's not particularly sensitive but best to remove it.

**Review by obelisk333 — CHANGES_REQUESTED** [review]
I agree with Jason's comments. Please remove sensitive info ASAP. Once approved this will need to be "squashed and merged" to ensure that these sensitive details do not become part of the version history.

**Review by obelisk333 — APPROVED** [review]
Looks fine.

**Pull request merged**